data/navigation_dataset/examples/objects_in_groups/example_0047.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<root>
+  <instruction text="the second and the third chair in the row of chairs aside the robot"/>
```

Comment:
the original dcg corpus had an example for and but this is the first time in ADCG